File: Libro.java
public class Libro {
    private static int nextId = 1;
    private int id;
    private String nombre;
    private String autor;
    private boolean rentado;

    public Libro(String nombre, String autor) {
        this.id = nextId++;
        this.nombre = nombre;
        this.autor = autor;
        this.rentado = false;
    }

    public int getId() {
        return id;
    }

    public String getNombre() {
        return nombre;
    }

    public String getAutor() {
        return autor;
    }

    public boolean isRentado() {
        return rentado;
    }

    public void setRentado(boolean rentado) {
        this.rentado = rentado;
    }
}

File: Libro.py
class Libro:
    next_id = 1

    def __init__(self, nombre, autor):
        self.id = Libro.next_id
        Libro.next_id += 1
        self.nombre = nombre
        self.autor = autor
        self.rentado = False

    def get_id(self):
        return self.id

    def get_nombre(self):
        return self.nombre

    def get_autor(self):
        return self.autor

    def is_rentado(self):
        return self.rentado

    def set_rentado(self, rentado):
        self.rentado = rentado